autopush/tests/test_endpoint.py

```diff
@@ -869,19 +873,18 @@ def test_decipher_public_key(self):
         payload = {"aud": "https://pusher_origin.example.com",
                    "exp": int(time.time()) + 86400,
                    "sub": "mailto:admin@example.com"}
-        (_, crypto) = self._gen_jwt(header, payload)
-        crypto_key = utils.base64url_decode(crypto)
-
-        spki_header = ('0V0\x10\x06\x04+\x81\x04p\x06\x08*'
-                       '\x86H\xce=\x03\x01\x07\x03B\x00\x04')
-        eq_(decipher_public_key(crypto_key), crypto_key)
-        eq_(decipher_public_key('\x04' + crypto_key), crypto_key)
-        eq_(decipher_public_key(spki_header + crypto_key), crypto_key)
+        (_, crypto_key) = self._gen_jwt(header, payload)
```

Comment:
There's more than just this one present. Will fix this along with others in a later PR.